src/test/java/com/amazon/opendistroforelasticsearch/security/ssl/util/TLSUtilTests.java

```diff
@@ -72,7 +72,7 @@ public void testSSLUtilInvalidContentLength() {
             buffer.writeByte(TLS_MAJOR_VERSION);
             buffer.writeByte(TLS_MINOR_VERSION);
             //setting content length as 0
-            buffer.writeByte(0);
+            buffer.writeShort(0);
```

Comment:
This is a separate issue that needs to be fixed on master and backported to all branches.

Reply:
Can we get this PR merged first given the tight timeline? I'll make the same change in all of other branches.

Reply:
The PR for master branch: https://github.com/opendistro-for-elasticsearch/security/pull/927

Reply:
No, there is no release that this PR blocks.